modules/oc-mirror-command-reference-v2.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-command-reference-v2_{context}"]
+= Command reference for V2
+
+The following tables describe the `oc mirror` subcommands and flags for V2:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
```

Comment:
These subcommands are not implemented yet as far as I know

Reply:
Oh, should I remove them then?

Reply:
@subhtk - Yes remove them for now, we plan to have these "commands" in a separate tool

Reply:
The only sub commands we should keep are delete and dry-run

Reply:
Ack..